### PR TITLE
OSDOCS#5228: Installing a three-node Azure cluster

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.13+

Issue:
This PR addresses [osdocs-5228](https://issues.redhat.com/browse/OSDOCS-5228).

Link to docs preview:

- [Installing a three-node cluster on Azure](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-three-node.html)
- Installing a cluster on Azure with customizations > [Creating the installation configuration file](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-initializing_installing-azure-customizations) (note under step 2)
- Installing a cluster on Azure using ARM templates > [Creating the installation configuration file](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra.html#installation-initializing_installing-azure-user-infra) (note under step 2)
- Installing a cluster on Azure using ARM templates > [Creating the Kubernetes manifest and Ignition config files](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-azure-user-infra) (warning before step 5)
- Installing a cluster on Azure using ARM templates > [Creating additional worker machines in Azure](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra.html#installation-creating-azure-worker_installing-azure-user-infra) (first note)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: installing/installing_azure/installing-azure-three-node.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a three-node cluster on Microsoft Azure. A three-node cluster consists of three control plane machines, which also act as compute machines. This type of cluster provides a smaller, more resource efficient cluster, for cluster administrators and developers to use for testing, development, and production.
+
+You can install a three-node cluster using either installer-provisioned or user-provisioned infrastructure.
+
+[NOTE]
+====
+Deploying a three-node cluster using an AWS Marketplace image is not supported.
```
Comment (jinyunma) [inline installing/installing_azure/installing-azure-three-node.adoc:15]: AWS -> Azure?

**Comment by jinyunma** [comment]
I could not find below sections from this PR's change, so put them here.
1. The note in [Configuring a three-node cluster](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-three-node.html#installation-three-node-cluster_installing-azure-three-node)
> Because application workloads run from control plane nodes, additional subscriptions are required, as the control plane nodes are considered to be compute nodes.

What does the additional subscriptions refer to?

2. the verification part in [same doc](https://56859--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-three-node.html#installation-three-node-cluster_installing-azure-three-node) 

> If you install a cluster on infrastructure that the installation program provisions, you can complete this verification step after the cluster is deployed.

For IPI, user could not verify the manifest file `cluster-scheduler-02-config.yml` after cluster is deployed, since the folder `manifests` (generated by running `openshift-install create manifests --dir <install_dir>`) are consumed when continuing to create ignition files or create cluster. Only way to check that , should be after creating manifests.

> If you install a cluster on infrastructure that you provision, you can complete this verification step after creating the Kubernetes manifest and Ignition configuration files.

sam as IPI, only check after creating manifests

File: installing/installing_azure/installing-azure-three-node.adoc (hunk 1 above)
Comment (mjpytlak) [reply installing/installing_azure/installing-azure-three-node.adoc]: Thank you. Updated.

**Comment by jinyunma** [comment]
Hi @mjpytlak I still doubt below statement in three-node installation doc
> After you create the Kubernetes manifest and Ignition config files, make sure that the spec.mastersSchedulable parameter is set to true in cluster-scheduler-02-config.yml file. You can locate this file in <installation_directory>/manifests.

Here `After you create the Kubernetes manifest and Ignition config files`,  refer to some section on other doc or installation steps.  file  `cluster-scheduler-02-config.yml ` checks can only be done after creating manifests. If after ignition config files, folder manifests is also consumed, you could not see that file any more.

**Comment by jinyunma** [comment]
other parts look good to me

**Pull request merged**